CefSharp.Wpf/ChromiumWebBrowser.cs

```diff
@@ -958,9 +961,48 @@ void IRenderWebBrowser.OnCursorChange(IntPtr handle, CursorType type, CursorInfo
             }
         }
 
+        private Visual GetParentWindow()
+        {
+            var current = VisualTreeHelper.GetParent(this);
+            while (current != null && !(current is Window))
+            {
+                current = VisualTreeHelper.GetParent(current);
+            }
+
+            return current as Window;
+        }
+
         void IRenderWebBrowser.OnImeCompositionRangeChanged(Range selectedRange, Rect[] characterBounds)
         {
-            OnImeCompositionRangeChanged(selectedRange, characterBounds);
```

Comment:
You've effectively broken OnImeCompositionRangeChanged for anyone overriding it.

I think this whole block can likely be moved into the handler for now.